docs/source/includes/running_choras.md

```diff
@@ -79,4 +81,5 @@ celery -A app.celery worker --loglevel=info -P eventlet
 ```
 
 ## Running the frontend
-Please refer to <https://github.com/choras-org/CHORAS/blob/dev/README.md#frontend-installation-open-a-new-command-window> for instructions on how to install and launch the frontend. <!-- TODO: link to the proper frontend documentation. -->
\ No newline at end of file
+
+Please refer to <https://github.com/choras-org/CHORAS/blob/dev/README.md#frontend-installation-open-a-new-command-window> for instructions on how to install and launch the frontend. <!-- TODO: link to the proper frontend documentation. -->
```

Comment:
```suggestion
Please refer to <https://github.com/choras-org/CHORAS/blob/dev/README.md#frontend-installation-open-a-new-command-window> for instructions on how to install and launch the frontend. 
```
Let's create an issue for this instead of this comment.